src/AppInstallerCLI/main.cpp

```diff
@@ -2,7 +2,11 @@
 // Licensed under the MIT License.
 #include <AppInstallerCLICore.h>
 
-int wmain(int argc, wchar_t const** argv)
+//int wmain(int argc, wchar_t const** argv)
+int wmain()
 {
-    return AppInstaller::CLI::CoreMain(argc, argv);
+    AppInstaller::CLI::TestCOMScenario();
+    return 0;
```

Comment:
You could just write a unit test to do this, and it might be something work checking in.